e2e/prerender/tests/tests.spec.js

```diff
@@ -108,10 +108,13 @@ test('charts should render once', async ({ page }) => {
 		return window[Symbol.for('chart renders')];
 	});
 
-	// 3 renders is what is expected (creation -> update -> (deletion) -> creation)
-	// double loading makes 4 renders, adding another delete/create in
-	// TODO: why is 3 renders expected
-	const expected = 7;
+	// 4 renders is what is expected (creation -> update x3)
+	// double loading makes 6 renders, adding another 2 updates in
+	// TODO: why are 3 updates expected
+	// - first is for initial render w/o chart props
+	// - second is for render w/ chart props
+	// - third is for ???, fourth is for ???
+	const expected = 10;
```

Comment:
This is getting smellier and smellier. What do we need to nail down what is expected here and remove these TODOs?